cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -24,6 +24,7 @@ function ExternalSite( element ) {
   var _dom = atomicHelpers.checkDom( element, BASE_CLASS );
   var _durationEl = _dom.querySelector( '.external-site_reload-container' );
   var _directEl = _dom.querySelector( '.external-site_proceed-btn' );
+  var _form = _dom.querySelector( 'form#proceed' );
```

Comment:
Though you might want to do the dom lookup before it's needed so that happens while the timer is going. Either should be fine.